src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -36,10 +44,11 @@ const PageLayoutWrapper = ({
 
   const scriptSwitchId = pathOr('', ['scriptSwitchId'], pageData);
   const renderScriptSwitch = pathOr(true, ['renderScriptSwitch'], pageData);
-  const isErrorPage = [404, 500].includes(status);
-  const pageType = isErrorPage
-    ? 'WS-ERROR-PAGE'
-    : path<string>(['metadata', 'type'], pageData);
+
+  const isErrorPage = ![200].includes(status) || !status;
```

Comment:
Should we now revert this back to the original?

```suggestion
  const isErrorPage = [404, 500].includes(status);
```

Reply:
what is the benefit of the original. Just in case we get some other codes that we don't want to assume is an error page? And also, what if there is no status for some reason?

Reply:
True. Let's leave your updated version - all pages that are working should have a status of 200, otherwise 💥